modules/face/include/opencv2/face/face_alignment.hpp

```diff
@@ -0,0 +1,306 @@
+/*
```

Comment:
In general for the documentation
- Make sure each explanation starts with a capital. Now it does sometimes but doesn't on other occasions making it not so clean in looks.
- Make sure, by going through the doc example website, that there are no more typo's, seen some small like ti-->to